internal/provider/user/models.go

```
@@ -101,6 +133,7 @@ func (m *userModel) FromDto(ctx context.Context, userDto sifflet.PublicUserGetDt
 	m.Email = types.StringValue(userDto.Email)
 	m.Role = types.StringValue(string(userDto.Role))
 	m.Permissions = permissionsList
+	m.AuthTypes, _ = types.SetValueFrom(ctx, types.StringType, userDto.AuthTypes)
```

Don't ignore the diagnostics (potential errors) here. 

In general, if you ignore an error, please include a comment explaining why.

```go
authTypes, diags := types.SetValueFrom(ctx, types.StringType, userDto.AuthTypes)
if diags.HasError() {
  return diags
}
m.AuthTypes = authTypes
```